curriculum/challenges/english/16-the-odin-project/top-basic-function-projects/top-basic-functions-exercise-c.md

```diff
@@ -0,0 +1,47 @@
+---
+id: 661e151f068359c3ccf2f4d7
+title: Basic Functions Exercise C
+challengeType: 1
+dashedName: top-basic-functions-exercise-c
+---
+
+# --description--
+
+Write a function, named `capatalize`, that takes a string as an argument and returns the string with the first letter capitalized.
+
+# --hints--
+
+You should have a function named `capatalize`.
+
+```js
+assert.strictEqual(capatalize('hello') , 'Hello');
```

Comment:
here we should just check that they created the function instead of if they completed it or not.

```suggestion
assert.isFunction(capitalize);
```